samples/grids/grid/editing-excel-style/src/index.tsx

```diff
@@ -1,146 +1,138 @@
-import React from 'react';
+import React, { useRef, useEffect } from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
 
-import { IgrGridModule } from 'igniteui-react-grids';
+import { IgrActiveNodeChangeEventArgs, IgrGridModule } from 'igniteui-react-grids';
```

Comment:
IgrActiveNodeChangeEventArgs isn't used - either remove the import or better yet - use the type for the event handler

Reply:
removed and updated with IgrActiveNodeChangeEventArgs